pkg/services/ngalert/ngalert.go

```diff
@@ -149,14 +149,10 @@ func (ng *AlertNG) init() error {
 	initCtx, cancelFunc := context.WithTimeout(context.Background(), 30*time.Second)
 	defer cancelFunc()
 
-	store := &store.DBstore{
-		Cfg:              ng.Cfg.UnifiedAlerting,
-		FeatureToggles:   ng.FeatureToggles,
-		SQLStore:         ng.SQLStore,
-		Logger:           ng.Log,
-		FolderService:    ng.folderService,
-		AccessControl:    ng.accesscontrol,
-		DashboardService: ng.dashboardService,
+	store, err := store.ProvideDBStore(ng.Cfg, ng.FeatureToggles, ng.SQLStore,
+		ng.folderService, ng.accesscontrol, ng.dashboardService)
+	if err != nil {
+		return err
```

Comment:
There is already the store registered in `wire`. I wonder if it would be better if we just accept it via ngalert.ProvideService. @grobinson-grafana, it looks like you introduced `store.ProvideDBStore`, can you think of any reasons why we did not use it here?

Also, this changes the logger, currently it is `ngalert` but will be `dbstore`.

Reply:
Changed it so the store is accepted via ngalert.ProvideService here: https://github.com/grafana/grafana/pull/67259/commits/e2755e5178c073a94052c68cbce2fdf609fa37cc. And also made the change to the logger 👍